package.json

```diff
@@ -114,7 +114,7 @@
     "ts-node": "10.7.0",
     "tslint": "^6.1.3",
     "tsx": "^3.12.7",
-    "typescript": ">=4.3.5 <5.4.0"
+    "typescript": ">=4.3.5 <5.4.0 || 5.3.1-rc"
```

Comment:
You also need to update the resolution below yeah?

Reply:
Waiting on https://github.com/microsoft/TypeScript/issues/56371.

Reply:
Ah, the CI...